test/jdk/java/lang/annotation/LoaderLeakTest.java

```diff
@@ -46,7 +77,7 @@ static void doTest(boolean readAnn) throws Exception {
         // URL[] path = { classes };
         // URLClassLoader loader = new URLClassLoader(path);
         ClassLoader loader = new SimpleClassLoader();
-        WeakReference<Class<?>> c = new WeakReference(loader.loadClass("C"));
+        WeakReference<Class<?>> c = new WeakReference<Class<?>>(loader.loadClass("C"));
         if (c.get() == null) throw new AssertionError();
         if (c.get().getClassLoader() != loader) throw new AssertionError();
```

Comment:
please use the messages which explain what went wrong, so when the failure happens, the person who analyze it won't have to open test code every time to just understand which of assertions was hit.